demosplan/DemosPlanCoreBundle/Addon/Registrator.php

```diff
@@ -66,6 +65,8 @@ private function refreshAddonsYaml(): void
             'addons' => $this->addons,
         ];
 
+        // TODO: add a "this is autogenerated" note to the file
```

Comment:
I did a quick search, but I have no idea how to include that in the dump. We might have to create the file first with the comment and then append the dump, which seems a bit too much trouble for a comment, I think.

Reply:
It's not that complicated, I just forgot about it. We just dump into a string and prepend the comment, then write the string to the file.